demo/index.html

```diff
@@ -88,6 +88,18 @@ <h1>Shadow DOM examples</h1>
           <auro-input disabled value="disabled" type="email" label="First name" helptext="Enter your first name" required id="disabledEmailTest01"></auro-input>
         </template>
       </demo-snippet>
+
+      <demo-snippet>
+        <template>
+          <auro-input id="format-ccWithIcon" format="credit-card" icon="true" label="Credit Card Number with Icon" required></auro-input>
```

Comment:
```suggestion
          <auro-input id="format-ccWithIcon" format="credit-card" icon label="Credit Card Number with Icon" required></auro-input>
```

HTML boolean attributes do not have a `true` or `false` value. 

`icon=""` === true
`icon="icon"` === true
`icon="false" === true

We can't assume that `icon="true"` is `true` because `icon="false"` will return true. Only the absence of the attribute is false.

https://html.spec.whatwg.org/multipage/common-microsyntaxes.html#boolean-attributes

This is not to be confused with Aria attributes that are not boolean, but support a [tristate](https://www.w3.org/TR/wai-aria-1.1/#valuetype_tristate) value.